IGraphics/IGraphicsStructs.h

```diff
@@ -2526,45 +2560,120 @@ struct IVStyle
-  , showLabel(showLabel)
-  , showValue(showValue)
-  , drawFrame(drawFrame)
-  , drawShadows(drawShadows)
-  , emboss(emboss)
-  , roundness(roundness)
-  , frameThickness(frameThickness)
-  , shadowOffset(shadowOffset)
-  , widgetFrac(widgetFrac)
-  , angle(angle)
-  , colorSpec(colors)
-  , labelText(labelText)
-  , valueText(valueText)
-  {
-  }
-  
+    : hideCursor(hideCursor)
+    , showLabel(showLabel)
+    , showValue(showValue)
+    , drawFrame(drawFrame)
+    , drawShadows(drawShadows)
+    , emboss(emboss)
+    , roundness(roundness)
+    , frameThickness(frameThickness)
+    , shadowOffset(shadowOffset)
+    , widgetFrac(widgetFrac)
+    , angle(angle)
+    , colorSpec(colors)
+    , labelText(labelText)
+    , valueText(valueText)
+  {
+  }
+
   /** Create a new IVStyle based on a list of colors, and defaults for the other elements
    * @param colors  */
   IVStyle(const std::initializer_list<IColor>& colors)
-  : colorSpec(colors)
-  {
-  }
-  
-  IVStyle WithShowLabel(bool show = true) const { IVStyle newStyle = *this; newStyle.showLabel = show; return newStyle; }
-  IVStyle WithShowValue(bool show = true) const { IVStyle newStyle = *this; newStyle.showValue = show; return newStyle; }
-  IVStyle WithLabelText(const IText& text) const { IVStyle newStyle = *this; newStyle.labelText = text; return newStyle;}
-  IVStyle WithValueText(const IText& text) const { IVStyle newStyle = *this; newStyle.valueText = text; return newStyle; }
-  IVStyle WithHideCursor(bool hide = true) const { IVStyle newStyle = *this; newStyle.hideCursor = hide; return newStyle; }
-  IVStyle WithColor(EVColor idx, IColor color) const { IVStyle newStyle = *this; newStyle.colorSpec.mColors[idx] = color; return newStyle; }
-  IVStyle WithColors(IVColorSpec spec) const { IVStyle newStyle = *this; newStyle.colorSpec = spec; return newStyle; }
-  IVStyle WithRoundness(float v) const { IVStyle newStyle = *this; newStyle.roundness = Clip(v, 0.f, 1.f); return newStyle; }
-  IVStyle WithFrameThickness(float v) const { IVStyle newStyle = *this; newStyle.frameThickness = v; return newStyle; }
-  IVStyle WithShadowOffset(float v) const { IVStyle newStyle = *this; newStyle.shadowOffset = v; return newStyle; }
-  IVStyle WithDrawShadows(bool v = true) const { IVStyle newStyle = *this; newStyle.drawShadows = v; return newStyle; }
-  IVStyle WithDrawFrame(bool v = true) const { IVStyle newStyle = *this; newStyle.drawFrame = v; return newStyle; }
-  IVStyle WithWidgetFrac(float v) const { IVStyle newStyle = *this; newStyle.widgetFrac = Clip(v, 0.f, 1.f); return newStyle; }
-  IVStyle WithAngle(float v) const { IVStyle newStyle = *this; newStyle.angle = Clip(v, 0.f, 360.f); return newStyle; }
-  IVStyle WithEmboss(bool v = true) const { IVStyle newStyle = *this; newStyle.emboss = v; return newStyle; }
+    : colorSpec(colors)
+  {
+  }
+
+  IVStyle WithShowLabel(bool show = true) const
+  {
+    IVStyle newStyle = *this;
+    newStyle.showLabel = show;
+    return newStyle;
+  }
```

Comment:
This is an issue, because it hasn't added a blank line when expanding these helper functions onto different lines

Reply:
yeah i couldn't work out what rule would influence that. Fixed up manually